Sage/carbon
# fix(profile): accessibility violation when no email is provided

fix #6754

### Proposed behaviour



When the `email` prop is not provided, the anchor element is not rendered. 
When the `text` prop is not provided, the span element is not rendered. 

### Current behaviour



When the `email` prop is not provided, the anchor element is still rendered causing an accessibility violation: "Links must have discernible text".
When the `text` prop is not provided, the span element is still rendered. 

### Checklist




#### QA


### Additional context



### Testing instructions



Go to test story for Profile, remove the `email` from props, see accessibility violation: "Links must have discernible text".
